index.html

```diff
@@ -0,0 +1,274 @@
+    </header>
+
+    <!-- hero-section-start -->
+    <!-- home-section-start -->
+    <section class="home-page">
+      <div class="page-container">
+        <div class="page-content">
+          <h1 class="page-title">
+            Launch Your <br />
+            Dev Career <br />
+            With Project- <br />
+            Based <br />
+            Coaching
+          </h1>
+          <p class="title-description">
+            showcase your skills with practical development experience and land
+            the coding career of your dreams
+          </p>
+          <div class="social-div">
+            <ul class="social-icons">
+              <li><i class="fab fa-youtube"></i> 2.3m+</li>
+              <li><i class="fab fa-instagram"></i> 3.5m+</li>
+            </ul>
+            <p class="icons-description">Global followers and subscribers</p>
+          </div>
+        </div>
+        <div class="img-div">
+          <img src="./images/hero.webp" alt="data" class="page-img" />
+        </div>
+      </div>
+    </section>
+
+    <!-- home-section-end -->
+
+    <!-- catalog-section-start -->
+    <section class="catalog-section" id="catalog">
+      <h2 class="catalog-title">Explore the Catalog</h2>
+      <hr class="catalog-line" />
+      <ul class="catalog-container">
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="lang" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+        <li class="catalog-description">
+          <img src="./images/language4.webp" alt="" />
+          <h3>Html / Css</h3>
+          <p>
+            Beginner-friendly Lessons-mastery-level Lessons-pro-level Lessons
+          </p>
+        </li>
+      </ul>
+      <a href="#" class="catalog-btn">join the tech wolrd</a>
+      <a href="#" class="catalog-text">see the whole program</a>
+    </section>
+
+    <!-- catalog-section-ends -->
+
+    <!-- student-review-start -->
+    <section class="partners" id="testimonial">
+      <h1 class="testimonial-title">Students Testimonial</h1>
+      <hr class="line" />
+      <div class="testimonials"></div>
```

Comment:
- [ ] Kindly adjust the margins and the size of the fonts of this section to match that of the design template as much as possible
Your design
![Capture2](https://user-images.githubusercontent.com/85095702/175773315-14ad1c42-3576-419c-ab17-96028060add9.jpg)
Behance template
![figma2](https://user-images.githubusercontent.com/85095702/175773345-56b844f2-c05a-4712-8e17-97335bce1f82.jpg)